### PR TITLE
refactor: use xref measurement instead of omics

### PR DESCRIPTION
**Pull request by hredestig**
adjust for changes in iloop

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,9 +13,11 @@ services:
       - PYTHONUNBUFFERED=1
       - PYTHONASYNCIODEBUG=1
       - ILOOP_TOKEN
-      - ILOOP_API=${ILOOP_API:-iloop-backend:80/api}
+      - ILOOP_API=${ILOOP_API:-http://iloop-backend:80/api}
     networks:
-      - iloop-net
+      iloop-net:
```
Comment (Midnighter) [inline docker-compose.yml]: When the https://github.com/biosustain/iloop/pull/73 gets merged (tomorrow morning), the network will be renamed to just `iloop`.

File: upload/checks.py [hunk 2]
```diff
@@ -13,17 +13,17 @@
 
 def load_identifiers(type):
     iloop = iloop_client(Default.ILOOP_API, Default.ILOOP_TOKEN)
-    return set(iloop.BiologicalIdentifier.subset(type=type))
+    return set(iloop.Xref.subset(type=type))
```
Comment (Midnighter) [inline upload/checks.py]: Prefer `frozenset`?

File: docker-compose.yml (hunk 1 above)
Comment (hredestig) [inline docker-compose.yml]: alright, needs to be fixed everywhere else as well..

File: upload/checks.py (hunk 2 above)
Comment (hredestig) [inline upload/checks.py]: Sure, done

**Comment by hredestig** [comment]
Good idea, done

**Pull request merged**